src/xpcc/communication/xpcc/backend/zeromq/reader.hpp

```diff
@@ -0,0 +1,177 @@
+// coding: utf-8
+/* Copyright (c) 2016, Roboterclub Aachen e. V.
```

Comment:
yes, we recently discovered, that this is non ideal

this is also what @dergraaf was referring to in his comment about updating the copyright information.